docs/electrum.md

```diff
@@ -0,0 +1,14 @@
+# SeedSigner Electrum seed phrase support
+
+SeedSigner supports loading of Electrum's Segwit seed phrases.  This is considered an Advanced feature that is disabled by default.  
+
+To load an Electrum Segwit seed phrase, first enable Electrum seed support in Settings -> Advanced -> Electrum seed support.  After this option is enabled, the user will now be able to enter an Electrum seed phrase by selecting "Enter Electrum seed" in the Load Seed screen.
+
+Some SeedSigner functionality is deliberately disabled when using an Electrum mnemonic:
+
+- BIP-85 child seeds
+	- Not applicable for Electrum seed types
+- SeedQR backups
+	- Since Electrum seeds are not supported by other SeedQR implementations, it would be dangerous to use SeedQR as a backup tool for Electrum seeds and is thus disabled
+- Custom derivations
```

Comment:
I can understand not doing this for bip85 because the user might NOT be expecting a bip39 mnemonic.  But wondering why no custom derivation for electrum seeds.

Reply:
Electrum doesn't use standard derivation paths, or let users edit them, they are basically hardcoded based on the seed type.